images/macos/provision/core/nvm.sh

```diff
@@ -11,18 +11,9 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/$VERSION/install.sh | bash
 if [ $? -eq 0 ]; then
```

Comment:
Please update the script description above

Reply:
Thank you. Done.